### DISCUSSION TITLE
Inverser "élémentaire" et "primaire"?

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1390] [Idées]
Je me demande s'il ne faudrait pas inverser "élémentaire" et "primaire" dans le menu déroulant "Secteurs d'activité" sur le formulaire de création d'une cantine. Qu'en penses-tu, @valeriemerle?

<img width="551" alt="Capture d’écran 2022-05-12 à 11 48 28" src="https://user-images.githubusercontent.com/93709421/168043695-db43b988-4828-4ff9-9593-a9735d8bb00e.png">
<img width="768" alt="Capture d’écran 2022-05-12 à 11 48 57" src="https://user-images.githubusercontent.com/93709421/168043705-4f080efe-ad26-4dcf-b2cd-6a602944ba6c.png">

**Comment by valeriemerle** [comment]
oui tu as raison, j'ai fais tjs l'erreur. Je fais la modif de suite.
